public/sass/components/_panel_logs.scss

```diff
@@ -93,36 +110,56 @@
 }
 
 .logs-row__level {
-  background-color: transparent;
   position: relative;
-  width: 3px;
-  padding: 0 !important;
+  // width: 12px;
+
+  &::after {
+    content: '';
+    display: block;
+    position: absolute;
+    top: 1px;
+    bottom: 1px;
+    width: 3px;
+    background-color: $gray-2;
```

Comment:
This no longer matches the log graph label color for `Unknown` log levels.

Reply:
ah, yes. where is that set? needs to be theme aware probably